src/LercLib/BitStuffer2.cpp

```diff
@@ -470,12 +479,27 @@ bool BitStuffer2::BitUnStuff(const Byte** ppByte, size_t& nBytesRemaining, vecto
   if (numElements == 0)

```

Comment:
if `maxElementCount` is made a `size_t` all uses of `numElements` should also be a `size_t`.  Ensure 64-bit compatibility and consistent behavior with 32-bit behaviors.